src/logic/calculate.js

```diff
@@ -0,0 +1,66 @@
+import operate from './operate';
+
+const isOperation = ope => ['X', '-', '+', '÷', '+/-', '%'].indexOf(ope) > -1;
```

Comment:
### [optional] 

Let's be more professional in our naming conversion, `ope` seems like a Nigeria(yoruba) name 😃 .
You could use index.
You can also change `X` to `x` so that X won't be bigger than the rest of the operators.